src/xmipp/libraries/py_xmipp/deepLearningToolkitUtils/utils.py

```diff
@@ -3,19 +3,21 @@
 
 BAD_IMPORT_MSG='''
 Error, tensorflow/keras is probably not installed. Install it with:\n  ./scipion installb deepLearningToolkit
-If gpu version of tensorflow desired, install cuda 8.0 or cuda 9.0
-We will try to automatically install cudnn, if unsucesfully, install cudnn and add to LD_LIBRARY_PATH
-add to SCIPION_DIR/config/scipion.conf
+If gpu version of tensorflow desired, modify  SCIPION_DIR/config/scipion.conf
```

Comment:
We should point user to online doc, so that we don't have to keep it updated on two places